web/src/stream_settings_ui.js

```diff
@@ -811,14 +811,16 @@ export function launch(section, left_side_tab, right_side_tab) {
             },
         });
         change_state(section, left_side_tab, right_side_tab);
+        setTimeout(() => {
+            if (!stream_settings_components.get_active_data().id) {
+                if (section === "new") {
+                    $("#create_stream_name").trigger("focus");
+                } else {
+                    $("#search_stream_name").trigger("focus");
```

Comment:
Search input is focussed currently on main as well, but might be fine to have this inside `setTimeout` as well.

Reply:
So, should I update the code for this or the now one is fine ?

Reply:
Fine for now.We can wait for feedback from Tim.